src/ledger/rocksdb.zig

```diff
@@ -70,6 +90,8 @@ pub fn RocksDB(comptime column_families: []const ColumnFamily) type {
             };
         }
 
+        // QUESTION/NIT: What about naming this close? To be symmetric with open.
```

Comment:
agreed - init() would be better to be consistent -- can add a doc comment explaining how it opens the db